qiskit/ignis/experiments/calibration/instruction_data/gate_and_schedules_definition.py

```diff
@@ -55,243 +78,196 @@ def from_backend(cls, backend: BaseBackend) -> 'InstructionsDefinition':
         Returns:
             New InstructionSet instance.
         """
-        parameter_library = dict()
-        channel_qubit_map = dict()
+        # TODO implement this
+        # we need a logic to convert Schedule into database code.
+        # Schedule doesn't have context. Thus we need to infer the context.
+
+        raise NotImplementedError
```

Comment:
What was the issue with the current implementation?

Reply:
We don't have programming model to generate the code. We may be able to take `Schedule`, but it doesn't have context (all alignment contexts are removed and expressed with fixed t0 and duration). I want to avoid making a clone of schedule builder for ignis.

Reply:
Can we then try and envision how this class will be instantiated? Ideally we should be able to provide the user with this class populated with defaults. Is it still possible to do this from the `backend` object or will this have to be done elsewhere? This also touches on the issue that the intent of the pulses is not in the `InstMap`.